### DISCUSSION TITLE
[Feature] Public Immich with 1+ millions assets

### DISCUSSION THREAD

**Discussion by mingqlin** [immich-app/immich#7173] [Feature Request]
### The feature

This is just a recommendation, I would suggest the team create a public immich demo (a separate one or the same one we already have) with 1+ millions assets, it will help developers load test their application.

I have 800,000+ photos and videos, at this moment, immich both the website and mobile are unusable. Really love Immich, I hope it could work with multi-millions assets, performance should stay the same no matter how many assets it has

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is definitely something we would like to do. The main issue is in sourcing that many assets that have reasonable quality exif data and permission for us to use.
